spec/models/customer_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should have_many(:items).through(:customers_items) }
```

Comment:
Also want to test ` it { should have_many(:customer_items) }` on the line above this one, testing each piece of the association.